ind.ie.Gnomit.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>ind.ie.Gnomit.desktop</id>
+  <name>Gnomit</name>
+  <developer_name>Aral Balkan (Ind.ie)</developer_name>
+  <update_contact>hello@ind.ie</update_contact>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-or-later</project_license>
+  <summary>Git commit message editor</summary>
+	<description>
+	  <p>Gnomit is a simple Git commit message editor for Gnome, inspired by the excellent Komet app on macOS by Mayur Pawashe.</p>
+	  <p>Features:</p>
+	  <ul>
+	    <li>Highlights overflow of subject line when it exceeds 69 characters.</li>
+      <li>Inserts empty line between subject line and the rest of the message.</li>
+      <li>Has spell checking.</li>
+      <li>Select All selects only your commit message, not the Git commit comment.</li>
+      <li>Displays the project folder and branch in the window header.</li>
+      <li>The Git Commit comment is not editable.</li>
+	  </ul>
+	</description>
+  <url type="homepage">https://github.com/indie-mirror/gnomit</url>
+  <url type="bugtracker">https://github.com/indie-mirror/gnomit/issues</url>
+  <url type="donation">https://ind.ie/fund/</url>
+  <url type="help">https://forum.ind.ie/</url>
+  <categories>
```

Comment:
put these as categories in the desktop file instead, they'll get automatically copied into here in the build process

Reply:
Thanks, Ned. I’ll try that and include it in the next build. By the way, is this documented anywhere? I cobbled together these files from bits and pieces and examples I could find in a multitude of places.

Reply:
Ah, I see what happened. I followed the instructions on the [Generic Component](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#sect-Metadata-GenericComponent) page of the Appstream documentation instead of the [Desktop Applications](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html) section.

Reply:
Fixed in https://source.ind.ie/gnome/gnomit/gjs/commit/e7bae7c336b61af2d564f6885be2066be47dbedc and will be in the next push to this PR.